quickwit/quickwit-storage/src/lib.rs

```diff
@@ -320,9 +327,14 @@ pub(crate) mod test_suite {
         test_write_and_delete(storage)
             .await
             .context("write_and_delete")?;
-        test_write_and_bulk_delete(storage)
-            .await
-            .context("write_and_bulk_delete")?;
+        // Fake GCS Server doesn't support bulk delete correctly.
+        // ref: <https://github.com/fsouza/fake-gcs-server/issues/1443>
+        #[cfg(not(feature = "google"))]
```

Comment:
Let's not do the check using the compiler flag, but instead dynamically inspect the storage.
As is, your code is removing this test for azure too in our CI.

(these test are running in the confusingly named "coverage CI" that is triggered after each merge to main. That build takes a bit more time than the regular PR CI and includes azure etc.)

Reply:
Addresed, please review again